index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="logo"><img src="./images/logo.png" alt="My logo"></div>
```

Comment:
- [ ] You've designed this section really well. It really well matches the Figma template 👏. Although please implement this logo text to have an `a` link so that clicking it takes you to the home page. If you notice a lot of website clicking the logo takes you back to the homepage. Please implement this feature.
something like this would make sense. 
```suggestion
        <a class="logo"><img src="./images/logo.png" alt="My logo"></a>
```